server/opts.go

```diff
@@ -226,6 +231,7 @@ type Options struct {
 	JetStreamDomain       string            `json:"-"`
 	JetStreamExtHint      string            `json:"-"`
 	JetStreamKey          string            `json:"-"`
+	JetStreamLimits       JSLimitOpts       `json:"-"`
```

Comment:
Would not have added json tags with newer additions. I added a note long time ago at the top of Options:
```
// NOTE: This structure is no longer used for monitoring endpoints
// and json tags are deprecated and may be removed in the future.
```

Reply:
removed